packages/ckeditor5-utils/tests/collection.js

```diff
@@ -148,8 +148,31 @@ describe( 'Collection', () => {
 	} );
 
 	describe( 'add()', () => {
-		it( 'should be chainable', () => {
-			expect( collection.add( {} ) ).to.equal( collection );
+		it( 'should proxy its calls to addMany', () => {
```

Comment:
I don't think that we need to test it - it's an implementation detail and might change.

Reply:
Resolved in e13db78.